stellarphot/core.py

```diff
@@ -1106,10 +1106,10 @@ def from_vizier(
         Parameters
         ----------
 
-        header_or_center : FITS header or `astropy.coordinates.SkyCoord`
-            Either a FITS header with WCS information or a `SkyCoord` object.
-            The center of the frame or the input coordinate is the center
-            of the cone search.
+        location : `astropy.coordinates.SkyCoord`, `astropy.wcs.WCS`, or FITS header or
```

Comment:
Might want to edit some more, this documentation is not grammatically correct (I suspect the last 'or' in line 1109 should not be there).  Also extra space between words 'or' and 'a' at beginning of line 1111.  Honestly, I think the last sentence could be clearer that if you entered SkyCoord it represents center of frame or otherwise the center of the frame is extrapolated from the WCS information provided, just to clarify where the information is obtained.  But you know me, I am wordy. :)

Reply:
Good idea, done!